assets/queries/cloudFormation/docdb_cluster_master_user_password_rule/test/positive.yaml

```diff
@@ -0,0 +1,88 @@
+Resources:
+  NewAmpApp:
+    Type:  AWS::DocDB::DBCluster
+    Properties:
+      AvailabilityZones:
+        - String
+      BackupRetentionPeriod: Integer
+      DBClusterIdentifier: String
+      DBClusterParameterGroupName: String
+      DBSubnetGroupName: String
+      DeletionProtection: true
+      EnableCloudwatchLogsExports:
+        - String
+      EngineVersion: String
+      KmsKeyId: String
+      MasterUsername: admin
+      MasterUserPassword: 'asDjskjs73!!'
+      Port: Integer
+      PreferredBackupWindow: String
+      PreferredMaintenanceWindow: String
+      SnapshotIdentifier: String
+      StorageEncrypted: true
+      VpcSecurityGroupIds:
+        - String
+---
+Parameters:
+  ParentMasterPassword:
+    Description: 'Password'
+    Type: String
+    Default: 'asDjskjs73!'
+  ParentMasterUsername:
+    Description: 'username'
+    Type: String
+    Default: 'username!'
+Resources:
+  NewAmpApp-1:
+    Type: AWS::DocDB::DBCluster
+    Properties:
+      AvailabilityZones:
+        - String
+      BackupRetentionPeriod: Integer
+      DBClusterIdentifier: String
+      DBClusterParameterGroupName: String
+      DBSubnetGroupName: String
+      DeletionProtection: Boolean
+      EnableCloudwatchLogsExports:
+        - String
+      EngineVersion: String
+      KmsKeyId: String
+      MasterUsername: String
+      MasterUserPassword: !Ref ParentMasterPassword
+      Port: Integer
+      PreferredBackupWindow: String
+      PreferredMaintenanceWindow: String
+      SnapshotIdentifier: String
+      StorageEncrypted: Boolean
+      VpcSecurityGroupIds:
+        - String
+---
+Parameters:
+  ParentMasterUsername:
+    Description: 'username'
+    Type: String
+    Default: 'username!'
+Resources:
+  NewAmpApp-03:
+    Type:  AWS::DocDB::DBCluster
+    Properties:
+      AvailabilityZones:
+        - String
+      BackupRetentionPeriod: Integer
+      DBClusterIdentifier: String
+      DBClusterParameterGroupName: String
+      DBSubnetGroupName: String
+      DeletionProtection: true
+      EnableCloudwatchLogsExports:
+        - String
+      EngineVersion: String
+      KmsKeyId: String
+      MasterUsername: admin
+      MasterUserPassword: 'asDjskjs73!!'
+      Port: Integer
+      PreferredBackupWindow: String
+      PreferredMaintenanceWindow: String
+      SnapshotIdentifier: String
+      StorageEncrypted: true
+      VpcSecurityGroupIds:
+        - String
```

Comment:
[As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline